<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="0" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-02-19T14:06:15 IST" name="Default suite" finished-at="2023-02-19T14:06:47 IST" duration-ms="32413">
    <groups>
    </groups>
    <test started-at="2023-02-19T14:06:15 IST" name="Default test" finished-at="2023-02-19T14:06:47 IST" duration-ms="32413">
      <class name="Module1_Home_Test_Classes.TestNG_TestClass">
        <test-method is-config="true" signature="OpenBrowser()[pri:0, instance:Module1_Home_Test_Classes.TestNG_TestClass@6b927fb]" started-at="2023-02-19T14:06:15 IST" name="OpenBrowser" finished-at="2023-02-19T14:06:24 IST" duration-ms="9264" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OpenBrowser -->
        <test-method is-config="true" signature="loginToApp()[pri:0, instance:Module1_Home_Test_Classes.TestNG_TestClass@6b927fb]" started-at="2023-02-19T14:06:24 IST" name="loginToApp" finished-at="2023-02-19T14:06:37 IST" duration-ms="12867" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginToApp -->
        <test-method signature="TC1()[pri:1, instance:Module1_Home_Test_Classes.TestNG_TestClass@6b927fb]" started-at="2023-02-19T14:06:37 IST" name="TC1" finished-at="2023-02-19T14:06:38 IST" duration-ms="1117" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "org.apache.poi.ss.usermodel.Row.getCell(int)" because the return value of "org.apache.poi.ss.usermodel.Sheet.getRow(int)" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "org.apache.poi.ss.usermodel.Row.getCell(int)" because the return value of "org.apache.poi.ss.usermodel.Sheet.getRow(int)" is null
at Library_Files.Utility_Class.getDataFromExcelSheet(Utility_Class.java:64)
at Module1_Home_Test_Classes.TestNG_TestClass.TC1(TestNG_TestClass.java:67)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC1 -->
        <test-method is-config="true" signature="LogoutFromApp(org.testng.ITestResult)[pri:0, instance:Module1_Home_Test_Classes.TestNG_TestClass@6b927fb]" started-at="2023-02-19T14:06:38 IST" name="LogoutFromApp" finished-at="2023-02-19T14:06:40 IST" duration-ms="1893" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=TC1 status=FAILURE method=TestNG_TestClass.TC1()[pri:1, instance:Module1_Home_Test_Classes.TestNG_TestClass@6b927fb] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LogoutFromApp -->
        <test-method is-config="true" signature="loginToApp()[pri:0, instance:Module1_Home_Test_Classes.TestNG_TestClass@6b927fb]" started-at="2023-02-19T14:06:40 IST" name="loginToApp" finished-at="2023-02-19T14:06:43 IST" duration-ms="3042" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginToApp -->
        <test-method signature="VerifyTitle()[pri:2, instance:Module1_Home_Test_Classes.TestNG_TestClass@6b927fb]" started-at="2023-02-19T14:06:43 IST" name="VerifyTitle" finished-at="2023-02-19T14:06:43 IST" duration-ms="3" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at Module1_Home_Test_Classes.TestNG_TestClass.VerifyTitle(TestNG_TestClass.java:81)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyTitle -->
        <test-method is-config="true" signature="LogoutFromApp(org.testng.ITestResult)[pri:0, instance:Module1_Home_Test_Classes.TestNG_TestClass@6b927fb]" started-at="2023-02-19T14:06:43 IST" name="LogoutFromApp" finished-at="2023-02-19T14:06:47 IST" duration-ms="3925" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=VerifyTitle status=FAILURE method=TestNG_TestClass.VerifyTitle()[pri:2, instance:Module1_Home_Test_Classes.TestNG_TestClass@6b927fb] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LogoutFromApp -->
        <test-method is-config="true" signature="CloseBrowser()[pri:0, instance:Module1_Home_Test_Classes.TestNG_TestClass@6b927fb]" started-at="2023-02-19T14:06:47 IST" name="CloseBrowser" finished-at="2023-02-19T14:06:47 IST" duration-ms="248" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CloseBrowser -->
      </class> <!-- Module1_Home_Test_Classes.TestNG_TestClass -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
